parler_tts/modeling_parler_tts.py

```diff
@@ -1948,10 +2030,11 @@ def generate(
             inputs, generation_config.bos_token_id, model_kwargs
         )
         batch_size = input_ids.shape[0] // self.num_codebooks
+        kwargs_has_attention_mask = model_kwargs.get("attention_mask", None) is not None
```

Comment:
Even though we don't ever use `ParlerTTSForCausalLM`, have you tested it still ? No worries if not

Reply:
Not tested at all, I have not propagated the changes to ParlerTTSForCausalLM.